exercises/javascript-modal/main.js

```diff
@@ -0,0 +1,21 @@
+var $openModal = document.querySelector('.openModal');
+var $modal = document.querySelector('.modalHidden');
+var $noButton = document.querySelector('.noButton');
+var $modalContainer = document.querySelector('.modalContainerHidden');
+var isOpen = false;
+function open(event) {
+  if (isOpen === false) {
+    $modal.className = 'modalOpen';
+    $modalContainer.className = 'modalContainerOpen';
+    isOpen = true;
+  }
+}
+$openModal.addEventListener('click', open);
+function close(event) {
+  if (isOpen === true) {
+    $modal.className = 'modalHidden';
+    $modalContainer.className = 'modalContainerHidden';
+    isOpen = false;
+  }
```

Comment:
In this case, the boolean isn't really adding anything to the functionality you've made. If the button with the `open` event handler is clicked, we always want to open the modal, and if the button with the 1close` event handler is clicked, we always want to close the modal. You can just do the work for that situation without needing a boolean flag to check if you need to.